authz/client/src/main/java/org/keycloak/authorization/client/resource/PermissionResource.java

```diff
@@ -237,13 +237,43 @@ public void update(final PermissionTicketRepresentation ticket) {
         if (ticket.getId() == null) {
             throw new IllegalArgumentException("Permission ticket must have an id");
         }
-        Callable callable = new Callable() {
+        Callable<Void> callable = new Callable<Void>() {
             @Override
-            public Object call() throws Exception {
-                http.<List>put(serverConfiguration.getPermissionEndpoint()+"/ticket")
+            public Void call() throws Exception {
+                http.<Void>put(serverConfiguration.getPermissionEndpoint()+"/ticket")
                         .json(JsonSerialization.writeValueAsBytes(ticket))
                         .authorizationBearer(pat.call())
-                        .response().json(List.class).execute();
+                        .response()
+                        .execute();
+                return null;
+            }
+        };
+        try {
+            callable.call();
+        } catch (Exception cause) {
+            Throwables.retryAndWrapExceptionIfNecessary(callable, pat, "Error updating permission ticket", cause);
+        }
+    }
+
+    /**
+     * Deletes a permission ticket.
+     *
+     * @param ticket the permission ticket
+     */
+    public void delete(final PermissionTicketRepresentation ticket) {
```

Comment:
Instead of passing the ticket representation we can just use the ID (String) as the argument for this method.

Reply:
Thanks for the fast feedback, it is fixed now.